### ISSUE TITLE
Crashing with Apotheosis (at net.minecraft.enchantment.EnchantmentHelper.handler$zbp000$addAxeEnchantments(EnchantmentHelper.java:543) ~[?:?])

### ISSUE THREAD

**Issue by Coraline77447** [Fourmisain/AxesAreWeapons#3]
1.16.5
36.2.35
Axes Are Weapons: 1.5.1
Apotheosis: 4.8.99F

What happened?;

I was gathering spruce wood from a spruce wood mineshaft (Repurposed Structures) in a taiga biome that I'm living in to complete my home and I went exploring in the caves as a morphed bat and I stumbled apoun a dungeon BUT it was no ordinary dungeon...
It was a boss dungeon from Apotheosis and when I got too close to the boss spawner the game crashed.

What might be happening behind the scenes is that one of your mixins isn't working corectly or is definitely broken since Apotheosis calls to a vanilla class that one of your mixins calls too which caused the game to crash due to how the code that said mixin is using was made or that said mixin is broken.

Inconclusion; 

Apotheosis might have helped Axes Are Weapons to improve a bit of the code as of the 1.16 version BUT doesn't mean that the issue is present in the 1.18 versions as well that needs to be fixed.
If I were to test both mods out in 1.18 how much do you wanna bet that I'll run into the same problem like how I have in 1.16 btw?

I've reported the crash to Shadows in his discord about this and he says that a mixin is causing the problem; 
(**_.handler$zbp000$addAxeEnchantments_**)

Even tho both the crash report and the log are "screamin" Apotheosis but in reality it's Axes Are Weapons's fault for the crash since Axes Are Weapons is the catalyst that started it all while apotheosis was practically just a innocent bystander that got caught along with A.A.W (Axes Are Weapons), So I'm gonna temporarily uninstall A.A.W 1.16.5 till this issue is resolved...

Here's the crash report and the log:

https://pastebin.com/muKcK3Vd

[latest.log](https://github.com/Fourmisain/AxesAreWeapons/files/8838505/latest.log)

**Comment by Fourmisain** [comment]
This took a while to figure out.

This happens when an Apotheosis Boss Mob with an axe spawns.
The reason is that Apotheosis registers a [special enchantment](https://github.com/Shadows-of-Fire/Apotheosis/blob/c50f2d3b867e5b3bc85173b2231e3eb082f95bf7/src/main/java/shadows/apotheosis/ench/enchantments/InertEnchantment.java#L10) without a set category - not quite sure what's up with that, it seems to be unobtainable/unusable through normal means, so maybe it's some sort of placeholder.

Either way, this is a really easy fix on my side (I used `.equals()` when I could have just tested the category via `==` since it's an enum) and while I think Apotheosis' behavior might cause issues with other mods, I'm not certain it's really considered a bug, so I probably won't report it on their side.

**Comment by Fourmisain** [comment]
Getting late so here's a [quick release](https://github.com/Fourmisain/AxesAreWeapons/releases/tag/1.5.2) for Forge 1.16.

**Comment by Fourmisain** [comment]
k, releases are up!
(I really need to automate that process)

**Comment by Coraline77447** [comment]
I've just downloaded both the 1.16 and 1.18 version and will kinda be testing while playing.

Btw sometimes Apotheosis might spawn in one of their bosses from a boss spawner with a "special" weapon which may be an axe that's enchanted and may have a affix attached to it.
So when I got too close to one of those boss spawners and it spawned in despite the game's difficulty being in peaceful at the time which was probably wielding a axe which might have had a weapon enchantment applied to it. 
Also since I was in peaceful difficulty the boss mob would've despawned right after spawning in.

The Apotheosis bosses are just regular mobs (zombie, husk, skeleton, and stray) with infinite fire resistance applied to them while having mad health and attack damage and wearing any armor between iron and netherite (before 1.16 was just iron or diamond).
The ranged bosses most times will not have an enchanted bow so there's no worry exactly but the melee bosses are the real threat since they might one shot you even with enchanted iron armor and might drop your hp down pretty low with enchanted diamond armor.
How I know this is from personal experience with one of those suckers which was a skeleton with a madly enchanted sword and just destroyed my face within seconds and had kept kickin my teeth in untill I finally killed him since he had mad hp and obviously had one shot capability and he was a fast mf too....

So for those types of enemies it's best to shot them with powerful ranged weapons and have a really powerful melee weapon but for the ranged ones use powerful melee, especially if they're equipped with diamond armor since they'll tank out the damage with weapons that does little to no damage to them.
